lbr/x86.py

```diff
@@ -37,11 +37,33 @@
 LOAD      = r"v?mov[a-z0-9]*\s+[^,]*\("
 LOAD_ANY  = r"[a-z0-9]+\s+[^,]*\("  # use is_mem_load()
 MOV       = r"v?mov"
-MOVS_ZX     = r"%s(s|zx)" % MOV
+MOVS_ZX   = r"%s(s|zx)" % MOV
 STORE     = r"\s+\S+\s+[^\(\),]+,"  # use is_mem_store()
-TEST_CMP  = r"(test|cmp).?\s"
-
-MEM_IDX = r"\((%[a-z0-9]+)?,%[a-z0-9]+,?(1|2|4|8)?\)"
+TEST_CMP  = r"(test|cmp).?\s" 
+cvt_suff  = ['sd2si', 'si2sd', 'si2ss', 'ss2si', 'tss2si']
```

Comment:
This part of the script is related to common instructions mnemonics, its better to write the V2IIV list elsewhere and not mix it in, probably write it at the end of the file.

Reply:
moved to bottom

Reply:
Ok, but please restore the TEST_CMP and MEM_IDX to their original place.
Also, cvt_suff and suff are used only in the V2II2V list so declare them right above it and not up there.